Before-Using-Agoric.md

```diff
@@ -0,0 +1,79 @@
+
+# Before Using Agoric Software
+
+Before working with the Agoric CLI, Zoe, and other Agoric tools and
+software, you need to install the following. 
+
+<table border="1">
+  <tbody>
+  <th><b>Step</b></th>
+  <th><b>Action</b></th>
+  <th><b>Explanation</b></th>
+  <tr>
+    <td><center>1</center></td>
+    <td><a href="https://nodejs.org/">Install Node.js</a>, version 10.16.0 or higher</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><center>2</center></td>
+    <td><a href="https://classic.yarnpkg.com/en/docs/install#mac-stable">Install Yarn 1</a></td>
+    <td>Yarn is a package manager for your code and lets developers
+  share code with others. Code is shared via a <i>package</i> (also
+  called a <i>module</i>) that contains all shared code and a
```

Comment:
Hmm, I think this is not quite right. Packages and modules are two different things - packages are made of modules.